e2e-tests/playwright/utils/Common.ts

```diff
@@ -54,13 +54,50 @@ export class Common {
 
   async loginAsGithubUser() {
     await this.logintoGithub();
-    await this.page.goto('/');
+    await this.page.goto('/', { waitUntil: 'networkidle' });
```

Comment:
The network might not always work or can take more time. I debugged and found that problem is there is a `MuiCircularProgress-root` loading bar that appears before the sign-in page loads. We have `await common.waitForLoad();` which waits until the loading bar is not visible, and you can pass a timeout as needed, with the default being 120 seconds.